packages/chat-component/src/components/chat-stage.ts

```diff
@@ -0,0 +1,38 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { styles } from '../styles/chat-stage.js';
+import './chat-avatar.js';
+export interface ChatStage {
+  pagetitle: string;
+  svgIcon: string;
+  isEnabled: boolean;
+}
+
+@customElement('chat-stage')
+export class ChatStageComponent extends LitElement {
+  static override styles = [styles];
+
+  @property({ type: String })
+  pagetitle = '';
+
+  @property({ type: String })
+  url = '';
+
+  @property({ type: Boolean })
+  isEnabled = false;
+
+  @property({ type: String })
+  svgIcon = '';
+
+  override render() {
+    if (!this.isEnabled) {
```

Comment:
if you do want to keep the component, the enabled should be in the component outside so that there is you either don't render the shadow dom at all or create one.

Reply:
Yes, I had revisited this for the avatar component but had not gotten to this one. Still pending.